docs/contributing/building.rst

```diff
@@ -81,6 +94,8 @@ It is an installer, similar to miniconda, which installs an environment with con
 Conan
 -----
 
+For the Conan build you must run, ``scripts/setup_dev_ubuntu.sh`` as this build depends on many system level packages.
```

Comment:
nit: (comma placement, wording) For ...  you must run ``scripts/setup_dev_ubuntu.sh``, as this build depends ....